README.md

```diff
@@ -1,2 +1,149 @@
-# leaderboard
-This project aims to make use of gitflow, async functions, webpack and hitting of API's
+# Leaderboard
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [👥 Collaborator](#collaborator)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Loeaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a project that aims to integrate promises and callback and async functions in the calling in the hitting of an api.
+
+<br>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">Linters</a></li>
+    <li><a href="https://reactjs.org/">Javascript</a></li>
```

Comment:
- [ ] Please provide the appropriate links to each technology. you have used the same link which is the link to `react` for each tech. I kindly suggest you provide the correct link so your readme looks professional :+1: :sparkles: